src/gui_common/GUICommon.cs

```diff
@@ -89,6 +89,23 @@ public static void SmoothlyUpdateBar(TextureProgress bar, float target, float de
         bar.Value = MathUtils.Lerp((float)bar.Value, target, weight, 1.0f / (float)bar.MaxValue);
     }
 
+    /// <summary>
+    ///   Loads a Texture from predefined GUI asset texture folder path.
+    /// </summary>
+    public static Texture LoadGuiAsset(string file)
```

Comment:
This loads a texture, so this should be named as loading a texture instead of a generic "asset"